examples/with-react-intl/pages/_app.js

```diff
@@ -22,7 +22,7 @@ export default class MyApp extends App {
     // Get the `locale` and `messages` from the request object on the server.
     // In the browser, use the same values that the server serialized.
     const { req } = ctx
-    const { locale, messages } = req || window.__NEXT_DATA__.props.pageProps
+    const { locale, messages } = req || window.__NEXT_DATA__.props
```

Comment:
Also, this shouldn't use `__NEXT_DATA__` it should use `pageProps`

Reply:
There are no `pageProps` at client in this method.
As I see the only way to get something in `_app.js` added by server into `req` object is to return this in his own `getInitialProps` and then read from `__NEXT_DATA__`